src/modules/get-background.js

```diff
@@ -0,0 +1,19 @@
+import changeBackground from './change-background';
+
+const getBackground = async (weatherType) => {
+  try {
+    const apiKey = process.env.IMG_KEY;
+    const response = await fetch(`https://pixabay.com/api/?key=${apiKey}&q=${weatherType}+sky&image_type=photo&pretty=true&editors_choice=true`);
+    const background = await response.json();
+    if (background.cod === '404') {
+      // failed
```

Comment:
* [ ] Let return a message instead of just writing a comment, this will help you in debugging. You could return a customized status code ie,
`let response = {};`
`response={status: 404};`